### DISCUSSION TITLE
Stopping charon-systemd daemon is not stopping ipsec.

### DISCUSSION THREAD

**Discussion by gowdamanil** [strongswan/strongswan#2215] [Q&A]
Hello @tobiasbrunner 

I have successfully established IPSec between 2 hosts but the problem is:
1. I have killed charon-systemd daemon using command **/usr/bin/killall charon-systemd**
Even after that I am seeing IPSec SA not being destroyed.
Also, ESP packets are still seen.

Is this expected or should i follow other step?

**Comment by tobiasbrunner** [comment]
Did you actually send a SIGTERM to the daemon (should be the default used by `killall`)? Or did you send a different signal? If the latter and the daemon was not properly terminated, the IPsec SAs and polices in the kernel obviously can't get removed.
